### ISSUE TITLE
Descent Randomizer

### ISSUE THREAD

**Issue by roncli** [CDarrow/DXX-Retro#104]
Randomizer is an option you select when you're on the level selection screen.  When selected, it will shuffle all the enemies and all the powerups (ensuring you can still find the keys needed to unlock a door).

**Comment by phozon** [comment]
hell yeah

**Comment by schoerg** [comment]
I think something like this was posted on the Descent Discord.

**Comment by roncli** [comment]
Yeah, there's a branch out there with my implementation of randomizer.  I wanted to go further with it, though, including randomizing/shuffling powerups and keys.  Never got that far, though.
